man/docker-run.1.md

```diff
@@ -508,6 +509,18 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Docker to label the content with a private unshared label.
 Only the current container can use a private volume.
 
+To control mount propagation property of volume one can use `:[r]shared`,
+`:[r]slave` or `:[r]private` propagation flag. Propagation property can
+be specified only for bind mounted volumes and not for internal volumes or
+volume driver volumes. For this to work source mount point (mount point where
```

Comment:
No, it is not /var/lib/dokcer/volumes/. This directory container docker internal volumes and we don't allow passing propagation flags for docker internal volumes.

So it is the source mount of source-dir. So say you have a directory /root/foo which you want to bind mount inside container `-v /root/foo:/root/bar`. Then one can run `df` on `/root/foo` to figure out what's the source mount point. If /root/foo itself is a mount point, then source mount point is /root/foo. Otherwise most likely `/` is source mount point as /root/foo is mounted under `/`